DoxygenHelp.md

```diff
@@ -22,27 +22,37 @@ If DOXYGEN_GENERATE_DOCUMENTATION is selected in the CMake GUI, CMake will autom
+- DOXYGEN_OPTIONAL_INCLUDE_EARLYBINDING: If selected (... and already built), this will include the selected IFC schemas in the documentation. Otherwise, the schemas will not be included in the documentation.
+- DOXYGEN_OPTIONAL_AUTO_OPEN_DOCUMENTATION: If selected, the index.html page of the documentation will automatically open. If not selected, you will have to find and open the generated documentation manually (see section below on building the documentation). 
 
 ### ConfigureDoxygen.cmake file
 If DOXYGEN_GENERATE_DOCUMENTATION is selected in CMake, CMake will include ConfigureDoxygen.cmake. This file includes all settings for the generation of doxygen documentation for the Open Infra Platform project. Changes to documentation settings should be made here. doxygen_add_docs() creates the GenerateDocumentation target. The up-to-date version of the file can be found here:
 
 https://github.com/tumcms/Open-Infra-Platform/blob/development/cmake/ConfigureDoxygen.cmake
 
-A tag that is more likely to be required to change:
-- DOXYGEN_EXCLUDE_PATTERNS: if you want to exclude certain source code directories or files from the documentation, you should add them here.
+#### ToDos.
+DOXYGEN_EXCLUDE_PATTERNS is a tag that is more likely to be required to change.
+It excludes certain source code directories or files from the documentation. In this project, the following directories including the following patterns are currently excluded:
+- */CurrentlyExcluded/* 
+- */cmake/*
+- */deploy/*
+- */Documentation/* 
+- */external/*
+- */testdata/*
+- */Tools/* 
+- */UserInterface/QtPropertyBrowser/*
 
 More settings to be added can be found here:
 http://www.doxygen.nl/manual/index.html
 
-Important: when adding settingsthem in CMake, always use DOXYGEN_ before the standard doxygen tag, e.g. the doxygen tag "PROJECT_BRIEF" has to be "DOXYGEN_PROJECT_BRIEF" in the ConfigureDoxygen.cmake file.
+Important: when adding settings in CMake, always use DOXYGEN_ before the standard doxygen tag, e.g. the doxygen tag "PROJECT_BRIEF" has to be "DOXYGEN_PROJECT_BRIEF" in the ConfigureDoxygen.cmake file.
 
 ## Building the documentation
 In Visual studio, build "OpenInfraPlatform.GenerateDocumentation". This target can be found in Visual Studio under OpenInfraPlatform/Commands.
-In your build directory under Doxymentation/html you will then find something called “index.html”. This is a link to the index page of the HTML documentation browser. 
-This index page of the documentation will be opened automatically if the build succeeded.
+If "DOXYGEN_OPTIONAL_AUTO_OPEN_DOCUMENTATION" was selected in the CMake GUI, tye index page of the documentation will be opened automatically if the build succeeded.
+Otherwise, the documentation has to be opened manually. In your build directory under Doxymentation/html you will find something called “index.html”. This is a link to the index page of the HTML documentation browser. 

```

Comment:
index.html is the "landing page" of the documentation.

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/104/commits/db191d9d975f8d80b167ef09875962d736ab5077